src/Fhir.Anonymizer.Tool/FilesAnonymizerForNdJsonFormatResource.cs

```diff
@@ -96,13 +114,24 @@ public async Task AnonymizeAsync()
                         Console.WriteLine($"[{stopWatch.Elapsed.ToString()}][tid:{args.CurrentThreadId}]: {completedCount} Process completed. {failedCount} Process failed. {consumeCompletedCount} Consume completed.");
                     };
 
-                    await executor.ExecuteAsync(CancellationToken.None, false, progress).ConfigureAwait(false);
+                    await executor.ExecuteAsync(CancellationToken.None, true, progress).ConfigureAwait(false);
                 }
 
+                // Rename file name after success process
+                File.Move(tempBulkResourceOutputFileName, bulkResourceOutputFileName);
+
                 Console.WriteLine($"Finished processing '{bulkResourceFileName}'!");
             }
         }
 
+        private string GetTempFileName(string pathFileName)
+        {
+            string fileName = Path.GetFileName(pathFileName);
+            string directory = Path.GetDirectoryName(pathFileName);
+
+            return Path.Combine(directory, $"temp_{fileName}");
```

Comment:
Should we add some guid here to avoid any clashes?

Reply:
Thanks, updated.